pkg/cmd/run.go

```diff
@@ -162,3 +204,24 @@ func controlplaneFunc() func(r *envoy.Runtime) {
 		return func(r *envoy.Runtime) {}
 	}
 }
+
+// Function verifies template parameters passed in getenvoy command line.
+// If verification is successful, it returns a string which must be added to
+// Envoy command line to invoke a proper config.
+func processTemplateArgs(flavor string, args []string, templateParams map[string]string, runtime *envoy.Runtime) (string, error) {
+	for _, envoyParam := range args {
+		if strings.HasPrefix(envoyParam, "--config") {
+			return "", fmt.Errorf("--templateArg and %s cannot be specified at the same time", envoyParam)
+		}
+	}
+	config, err := flavors.CreateConfig(flavor, templateParams)
+	if err != nil {
+		return "", err
+	}
+	// Save config in getenvoy directory
+	path, err := runtime.SaveConfig(flavor, config)
+	if err != nil {
+		return "", err
+	}
+	return "--config-path " + path, nil
```

Comment:
Why do we concatenate arguments into a single string instead of returning a string array ?

Does `Envoy` understand correctly `'--config-path /path/to/config'` as a single string ?

Reply:
Yes